src/mmrelay/matrix/sync_bootstrap.py

```
@@ -0,0 +1,1201 @@
+            else:
+                facade.logger.error(
+                    f"Unexpected login response: {type(response).__name__}"
+                )
+                facade.logger.error(
+                    "This may indicate a matrix-nio library issue or server problem."
+                )
+
+            await client.close()
+            return False
+
+    except NIO_COMM_EXCEPTIONS + (ssl.SSLError, OSError):
```

Ruff (0.15.9)

[warning] Consider `(*NIO_COMM_EXCEPTIONS, ssl.SSLError, OSError)` instead of concatenation

Replace with `(*NIO_COMM_EXCEPTIONS, ssl.SSLError, OSError)`

(RUF005)